### DISCUSSION TITLE
I have submitted the application for four days, but I have not received the email

### DISCUSSION THREAD

**Discussion by xiaobaguniang** [community/community#147059] [GitHub Education]
### Hi everyone,

I have submitted the application for four days, but I have not received the email.Is there a solution to speed up the approval process, or is there a problem with my application

**Comment by Astekin34** [comment]
Hi @xiaobaguniang, try checking the status of your application on https://education.github.com/discount_requests/application, If it's accepted it takes about 5 days to get the benefits depending on density of applications, I recently applied for education on the 10th of December and got the benefits three days later on 13th of December, I'm pretty sure if you did everything right you would've gotten it by now. And unfortunately there isn't a way to speed it up. Have a nice day.
